start/routes.ts

```diff
@@ -20,6 +20,32 @@
 
 import Route from '@ioc:Adonis/Core/Route'
 
+Route.get('/users', 'GetsController.getAllUsers')
+Route.post('/register', 'PostsController.createUser')
+Route.post('/login', 'PostsController.login')
+
+Route.group(() => {
+  Route.post('/logout', 'PostsController.logout')
+  Route.get('/user/profile', 'GetsController.getUserProfile')
+  Route.post('/user/profile', 'PostsController.createProfile')
+  Route.put('/user/profile', 'PutsController.updateProfile')
+  Route.delete('/user/profile', 'DeletesController.deleteUser')
```

Comment:
You can group routes which has same prefix. For instance, in our case  `/user/`

Refer: Route section of Adonisjs.

Reply:
Done